### DISCUSSION TITLE
My Repo

### DISCUSSION THREAD

**Discussion by richgarcia7525** [EchterAlsFake/xnxx_api#3] [Ideas]
Like I had mentioned I am still new at this and it never even occured to me to send you my repo and if you could and when you have time taking a look at it and maybe pointing out how i messed up your perfectly working repo into what i present to you now. thank you in advance

This is probably the last thing anyone would want to hear from me BUT...I was glancing over your other repos. is there a suggestion of any you think would be a good addition to a website? just asking.

**Comment by EchterAlsFake** [comment]
?
